var/spack/repos/builtin/packages/libxc/package.py

```diff
@@ -73,8 +73,8 @@ def setup_build_environment(self, env):
         # by Spack, otherwise we may end up with contradictory or invalid flags
         # see https://github.com/spack/spack/issues/17794
 
+        env.append_flags("CFLAGS", "-std=c99")
```

Comment:
```suggestion
        # https://gitlab.com/libxc/libxc/-/issues/430 (configure script does not ensure C99)
        env.append_flags("CFLAGS", self.compiler.c99_flag)
```

Reply:
@haampie: @LecrisUT applied your suggested and requested change, please approve.